### PR TITLE
security update for the dompdf/dompdf vulnerability

### PR DESCRIPTION
**Pull request by Treggats**
This pull request updates the `barryvdh/laravel-dompdf` dependency.

The updated package addresses the [recent dompdf/dompdf vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2022-0085). (CVE-2022-0085)

### CODE REVIEW

File: src/Invoice.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace LaravelDaily\Invoices;
 
-use Barryvdh\DomPDF\Facade as PDF;
+use Barryvdh\DomPDF\Facade\Pdf;
```
Comment (Treggats) [inline src/Invoice.php:5]: needed to be updated because of [upstream changes](https://github.com/barryvdh/laravel-dompdf/pull/899)

**Review by michael-rubel — APPROVED** [review]

**Comment by michael-rubel** [comment]
@PovilasKorop take a look, please ⬆️ 🙏

**Comment by mc0de** [comment]
@michael-rubel will do when schedule frees up a bit, busy atm

**Comment by yormy** [comment]
+1

**Comment by neopheus** [comment]
+1

**Comment by michael-rubel** [comment]
@mc0de it shouldn't take much time though. People live with a huge security issue because of using this package.

**Comment by Treggats** [comment]
> @mc0de it shouldn't take much time though. People live with a huge security issue because of using this package.

We've been using my fork at work, waiting until this gets merged and released. Haven't run into any issues as of yet.

**Comment by michael-rubel** [comment]
@Treggats Will take a look, thanks!

**Comment by Treggats** [comment]
@michael-rubel no problem, worked with Barry for a quick release on his part. So happy to contribute 👍

**Comment by mc0de** [comment]
@michael-rubel @Treggats will merge this later today

**Comment by mc0de** [comment]
@Treggats @michael-rubel @yormy @neopheus 

updated as 3.0.1 composer should fetch soon

and added as a new release @ GH https://github.com/LaravelDaily/laravel-invoices/releases/tag/3.0.1

**Pull request merged**